Sources/NSString+RemoveEmoji.swift

```diff
@@ -10,8 +10,8 @@ import Foundation
 
 private extension UInt32 {
     static let combiningEnclosingKeycap: UInt32 = 0x20E3
-    static let varidationSelector15: UInt32 = 0xFE0E
-    static let varidationSelector16: UInt32 = 0xFE0F
+    static let validationSelector15: UInt32 = 0xFE0E
+    static let validationSelector16: UInt32 = 0xFE0F
```

Comment:
The correct name is "variation" selector. It's my fault.

Reply:
Fixed :)